addons/calendar/static/src/views/attendee_calendar/attendee_calendar_controller.js

```diff
@@ -3,13 +3,17 @@
 import { CalendarController } from "@web/views/calendar/calendar_controller";
 import { useService } from "@web/core/utils/hooks";
 
+const { onWillStart } = owl;
```

Comment:
We have a module that exports owl, so we don't need to get the object from the window.
```suggestion
import { onWillStart } from "@odoo/owl";
```

Reply:
Thanks for the suggestion! Commited.